qiskit_ibm/jupyter/dashboard/dashboard.py

```diff
@@ -104,17 +104,17 @@ def _get_backends(self) -> None:
         """Get all the backends accessible with this account."""
 
         ibm_backends = {}
-        for pro in IBMProvider.providers():
-            pro_name = "{hub}/{group}/{project}".format(hub=pro.credentials.hub,
-                                                        group=pro.credentials.group,
-                                                        project=pro.credentials.project)
-            for back in pro.backends():
-                if not back.configuration().simulator:
-                    if back.name() not in ibm_backends.keys():
-                        ibm_backends[back.name()] = \
-                            BackendWithProviders(backend=back, providers=[pro_name])
+        for hgp in IBMProvider._get_hgps():
+            hgp_name = "{hub}/{group}/{project}".format(hub=hgp.credentials.hub,
+                                                        group=hgp.credentials.group,
+                                                        project=hgp.credentials.project)
+            for backend in hgp._backends.values():
+                if not backend.configuration().simulator:
+                    if backend.name() not in ibm_backends.keys():
+                        ibm_backends[backend.name()] = \
+                            BackendWithProviders(backend=backend, providers=[hgp_name])
                     else:
-                        ibm_backends[back.name()].providers.append(pro_name)
+                        ibm_backends[backend.name()].providers.append(hgp_name)
```

Comment:
If the idea is to get all backends accessible with the account, wouldn't it just be `provider.backends()`?